lib/resources/r4/related_person_patch.yaml

```diff
@@ -96,10 +107,10 @@ operations:
 
     note: >
       <ul>
-        <li>The <code>value.coding</code> must contain exactly one proprietary system.</li>
-        <li>The <code>value.coding</code> cannot contain a standard system.</li>
         <li>When specifying a `period`, the fields must include a time component with a timezone.</li>
         <li>Duplicate relationships may be added, but will only be represented once on the Resource.</li>
+        <li>Each `coding` can be either from the standard or proprietary system, but not both.</li>
+        <li>If the coding is from a proprietary system, only one code may be provided.</li>
```

Comment:
nit: Add spaces after \<li> and before \</li>

Reply:
Applies throughout this file.

Reply:
Changed https://github.com/cerner/fhir.cerner.com/pull/662/commits/300867b2eb8e04f2456f4a36f5c45cc93172f23b#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR110-R113